sketch/README.md

```diff
@@ -0,0 +1,243 @@
+# @magenta/sketch
+
+This JavaScript implementation of Magenta's sketch-rnn model uses [TensorFlow.js](https://js.tensorflow.org) for GPU-accelerated inference. `sketch-rnn` is a recurrent neural network model described in [Teaching Machines to Draw](https://research.googleblog.com/2017/04/teaching-machines-to-draw.html) and [A Neural Representation of Sketch Drawings](https://arxiv.org/abs/1704.03477).
+
+![Example Images](https://cdn.rawgit.com/tensorflow/magenta/master/magenta/models/sketch_rnn/assets/sketch_rnn_examples.svg)
+
+*Examples of vector images produced by this generative model.*
+
+## SketchRNN
+
+This document is an introduction on how to use the Sketch RNN model in Javascript to generate images.  The Sketch RNN model is trained on stroke-based vector drawings. The model implementation here is able to handle unconditional (decoder-only) generation of vector images.
+
+For more information, please read original [model](https://magenta.tensorflow.org/sketch_rnn) description and for the Python TensorFlow implementation.
+
+## Getting started
+
+In the .html files, we need to include `magentasketch.js`. Our example sketch are built with p5.js and stored in a file such as `sketch.js`, so we have also included p5 libraries here too. Please see this minimal example:
+
+```html
+<html>
+<head>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.1/p5.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.1/addons/p5.dom.js"></script>
+  <script src="magentasketch.js"></script>
```

Comment:
you should use the jsdeliver cdn link (see the music readme), although this won't work until we publish

Reply:
Cool, yeah I'll leave this in like this for now, and then change it to jsdeliver after we publish later. But thanks for the note.